### PR TITLE
Add accordion widget

### PR DESCRIPTION
**Pull request by komal7292**

### CODE REVIEW

**Review by rohitksp — APPROVED** [review]

File: komal/Accordion/src/App.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  }
+]
+
+export default  () => {
+  return(
+    <div>
+      <Accordion items={items} />
+    </div>
+  ) 
+}
```
Comment (nrchandan) [inline komal/Accordion/src/App.js]: Please verify if this is throwing a warning in the js console.

File: komal/Accordion/src/reportWebVitals.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (nrchandan) [inline komal/Accordion/src/reportWebVitals.js:13]: Might be a good idea to delete these files until we understand them better.

**Review by nrchandan — CHANGES_REQUESTED** [review]

File: komal/Accordion/src/App.js (hunk 1 above)
Comment (komal7292) [reply komal/Accordion/src/App.js]: yes, it is giving warning on js console. i will fix this code.

File: komal/Accordion/src/reportWebVitals.js (hunk 2 above)
Comment (komal7292) [reply komal/Accordion/src/reportWebVitals.js:13]: yes. you are right

Comment (komal7292) [reply komal/Accordion/src/reportWebVitals.js:13]: yes, you are right.

**Review by nrchandan — APPROVED** [review]

**Pull request merged**